features/pull_request.feature

```diff
@@ -812,3 +812,36 @@ BODY
       Error creating pull request: Temporary Redirect (HTTP 307)
       Refused to follow redirect to https://disney.com/mouse\n
       """
+
+  Scenario: Default message with --push
+    Given the text editor adds:
+      """
+      """
+    Given the GitHub API server:
+      """
+      post('/repos/mislav/coral/pulls') {
+        halt 400 if request.content_charset != 'utf-8'
+        assert :title => 'This is somewhat of a longish title that does not get wrapped & references #1234',
+               :body => nil
+        status 201
+        json :html_url => "the://url"
+      }
+      """
+    Given I am on the "master" branch pushed to "origin/master"
+    When I successfully run `git checkout --quiet -b topic`
+    Given I make a commit with message "This is somewhat of a longish title that does not get wrapped & references #1234"
```

Comment:
I think validating the message is important here, since the logic is different with `--push` than otherwise. In particular, `--push` should construct the message based on changes from `origin/master` to the local `head`, whereas without it the message is based on changes from `origin/master` to the remote `head`.